### PR TITLE
Fix links, spelling, and grammar in documentation

### PR DESCRIPTION
**Pull request by adelq**

### CODE REVIEW

File: gallery/Demo2-ENCODE2_ChIA-PET/README.md [hunk 1]
```diff
@@ -83,12 +85,4 @@ When you have built tracks in MariaDB, it's very easy to build a genome browser.
...
-You can read this [toturial](https://github.com/Zhong-Lab-UCSD/Genomic-Interactive-Visualization-Engine/blob/master/tutorials/1-knownCodeDataSource.md) to learn how to simply tweak the genome browser. Our [demo](https://chiapet.givengine.org/) is based on the tweaked [`give_chiapet.html`](./give_chiapet.html) HTML file.
-
-
-
-
-
-
-
-
+You can read this [tutorial](https://github.com/Zhong-Lab-UCSD/Genomic-Interactive-Visualization-Engine/blob/master/tutorials/1.2-html-tweak.md) to learn how to simply tweak the genome browser. Our [demo](https://chiapet.givengine.org/) is based on the tweaked [`give_chiapet.html`](./give_chiapet.html) HTML file.
```
Comment (caoxiaoyi03) [inline gallery/Demo2-ENCODE2_ChIA-PET/README.md]: LGTM

File: manuals/3.1-GIVE-Toolbox-usages.md [hunk 2]
```diff
@@ -140,7 +140,7 @@ bash config_host.sh [-r <give_root>] [-u <mysqlu>] [-p <mysqlp>] [-d <host_domai
...
 
-If -a is set as "true", it will ignore other arguments and output all the reference genomes and associated track groups and tracks. If -a is not set as "true", the 3 arguments -r, -g, -t will determine the output structure level. If only set -r, it will output all the track group names and associated track names in the -r specified reference genome database. If set both -r and -g, then it wll list all the track names of -r -g specified track group. If set all the -r -g -t arguments, it will output the detail info of -r -g -t specified track.
+If -a is set as "true", it will ignore other arguments and output all the reference genomes and associated track groups and tracks. If -a is not set as "true", the 3 arguments -r, -g, -t will determine the output structure level. If only set -r, it will output all the track group names and associated track names in the -r specified reference genome database. If set both -r and -g, then it will list all the track names of -r -g specified track group. If set all the -r -g -t arguments, it will output the detail info of -r -g -t specified track.
```
Comment (caoxiaoyi03) [inline manuals/3.1-GIVE-Toolbox-usages.md]: LGTM

File: manuals/4.1-bed.md [hunk 3]
```diff
@@ -100,7 +100,7 @@ The following settings of tracks are available for BED track:
 *   `longLabel` - More detailed description of the track. Will be shown in a
     future update.
 *   `priority` - The order of the track in the browser. A smaller value means the
-    the track will be shown in a higher location.
+    track will be shown in a higher location.
```
Comment (caoxiaoyi03) [inline manuals/4.1-bed.md]: LGTM

File: manuals/4.2-bigwig.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ The following settings of tracks are available for BED track:
 *   `longLabel` - More detailed description of the track. Will be shown in a
     future update.
 *   `priority` - The order of the track in the browser. Smaller value means the
-    the track will be shown in a higher location.
+    track will be shown in a higher location.
```
Comment (caoxiaoyi03) [inline manuals/4.2-bigwig.md]: LGTM

File: manuals/4.3.1-HiCtool.md [hunk 5]
```diff
@@ -30,7 +30,7 @@ To create a Fend object use the function **hifive.fend**:
 ```
 2. Creating a HiCData object
 
-**HiC dataset** (hdf5 format) created from a Fend file and mapped data in bam format. The original bam file has to be splitted into two bam files (_see preprocessing `<preprocessing_data.html#splitting-the-bam-file-to-separate-the-two-reads-in-a-pair>`_) each containing the information of one of the two mapped reads, and then they are passed to the function as elements of a list. The 'maxinsert' parameter (int.) is a cutoff for filtering paired end reads whose total distance to their respective restriction sites exceeds this value.
+**HiC dataset** (hdf5 format) created from a Fend file and mapped data in bam format. The original bam file has to be split into two bam files (_see preprocessing `<preprocessing_data.html#splitting-the-bam-file-to-separate-the-two-reads-in-a-pair>`_) each containing the information of one of the two mapped reads, and then they are passed to the function as elements of a list. The 'maxinsert' parameter (int.) is a cutoff for filtering paired end reads whose total distance to their respective restriction sites exceeds this value.
```
Comment (caoxiaoyi03) [inline manuals/4.3.1-HiCtool.md]: LGTM

File: manuals/4.3-interaction.md [hunk 6]
```diff
@@ -55,7 +55,7 @@ The following settings of tracks are available for BED track:
 *   `longLabel` - More detailed description of the track. Will be shown in a
     future update.
 *   `priority` - The order of the track in the browser. Smaller value means the
-    the track will be shown in a higher location.
+    track will be shown in a higher location.
```
Comment (caoxiaoyi03) [inline manuals/4.3-interaction.md]: LGTM

File: tutorials/3-GIVE-Toolbox.md [hunk 7]
```diff
@@ -42,9 +42,9 @@ The current version of GIVE-Toolbox includes 9 script tools. These tools can mee
...
 
-For utilizing GIVE-Toolbox with [different GIVE deployment approaches](2-deploy.md):
-- GIVE-Docker: GIVE-Toolbox is integrated in the GIVE-Docker. When you run a GIVE container and log into the container system (bash terminal), all the script tools can be directly executed with their names, which are located in system path `/usr/local/bin` (the container system). The `example_data` is in the `/tmp` path (the container system). 
-- Custom installation: You need to clone the GIVE GitHub repository and install it correctly . `GIVE-Toolbox` folder is included in the cloned GIVE repo. You can use these tools with their directory like `bash ~/GIVE/GIVE-Toolbox/add_track_bed.sh -u ...`. For convenience, you can set all these tools excutable (using `chmod +x` command to them) and copy them to a system path folder (such as `/usr/local/bin` or declare some path in your `.bashrc` file). Then you can directly run these tools.
+For utilizing GIVE-Toolbox with different GIVE deployment approaches:
```
Comment (caoxiaoyi03) [inline tutorials/3-GIVE-Toolbox.md]: LGTM

File: tutorials/Readme.md [hunk 8]
```diff
@@ -27,11 +27,11 @@ We documented both approaches of deployment in GIVE Tutorial 2.1 and 2.2. The fo
 
 ### [GIVE Tutorial 2.1: Local deployment of GIVE with GIVE-Docker](2.1-GIVE-Docker.md)
 
-This tutorial shows deployment of GIVE to your local machines using GIVE-Docker. This is the recommendated approach for deploying GIVE. No system configuration is required.  
+This tutorial shows deployment of GIVE to your local machines using GIVE-Docker. This is the recommended approach for deploying GIVE. No system configuration is required.
```
Comment (caoxiaoyi03) [inline tutorials/Readme.md]: LGTM

File: gallery/Demo1-captureHiC_promoter_contacts/README.md [hunk 9]
```diff
@@ -80,6 +80,4 @@ When you have built tracks in MariaDB, it's very easy to build a genome browser.
 </chart-controller>
 ```
 
-You can read this [toturial](https://github.com/Zhong-Lab-UCSD/Genomic-Interactive-Visualization-Engine/blob/master/tutorials/1-knownCodeDataSource.md) to learn how to simply tweak the genome browser. Our [demo](https://chic.givengine.org/) is based on the tweaked [`give_chic.html`](./give_chic.html) html file.
-
-
+You can read this [tutorial](https://github.com/Zhong-Lab-UCSD/Genomic-Interactive-Visualization-Engine/blob/master/tutorials/1.2-html-tweak.md) to learn how to simply tweak the genome browser. Our [demo](https://chiapet.givengine.org/) is based on the tweaked [`give_chiapet.html`](./give_chiapet.html) HTML file.
```
[the remark this answers is not in the document]
Comment (adelq) [reply gallery/Demo1-captureHiC_promoter_contacts/README.md]: My mistake, it's fixed now.

**Comment by adelq** [comment]
Commit updated with the fix, thanks!

**Pull request merged**